### PR TITLE
Scopes and Types 

### PR DESCRIPTION
**Pull request by leorcodes**
PR for a Unit test, will be committing the Integration later today

### CODE REVIEW

**Review by melyxlin — APPROVED** [review]

File: web/client/src/components/Credentials.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            </MDBCol>
+            <fieldset id="format">
+              <MDBRow>
+                <MDBCol>
+                  {" "}
+                  <Radio
+                    name="format"
+                    value="Bare"
+                    id="defaultGroupExample3"
+                    onChange={formatChange}
```
Comment (yousefa00) [inline web/client/src/components/Credentials.js]: You may be able to just have `onChange={setFormat}` here and for other cases where you use `onChange` with `formatChange` or `typeChange`

**Review by yousefa00 — APPROVED** [review]
Looks good, left one comment about state setter 🚀

**Review by melyxlin — APPROVED** [review]
Looks good to me.

File: web/client/src/components/Credentials.js (hunk 1 above)
Comment (leorcodes) [reply web/client/src/components/Credentials.js]: Yea I was able to do that, thanks!

**Pull request merged**